src/HonzaBotner.Discord.Services/Commands/VoiceCommands.cs

```diff
@@ -11,7 +10,7 @@
 namespace HonzaBotner.Discord.Services.Commands
 {
     [Group("voice")]
-    [Description("Commands to control custom voice channels.")]
+    [Description("Commands to control (and only works in) custom voice channels.")]
```

Comment:
What does custom voice channel means?

Reply:
Those are the channels that members can create by themselves. Temporary channels. See previous issues or config.

![image](https://user-images.githubusercontent.com/5287596/108262600-6bd9bf00-7165-11eb-8c8b-64f5b463ef11.png)